src/Moryx.Runtime.Kernel/Modules/ComponentContracts/IModuleFailureStrategy.cs

```diff
@@ -10,9 +10,5 @@ namespace Moryx.Runtime.Kernel
     /// </summary>
     internal interface IModuleFailureStrategy
```

Comment:
I think the entire interface is obsolote now

Reply:
Ok, I will remove it.

Reply:
Thanks

Reply:
Done